db/schema.rb

```diff
@@ -151,7 +152,7 @@
     t.string "magic_link_token"
     t.datetime "magic_link_token_sent_at"
     t.boolean "hide_in_reporting", default: false, null: false
-    t.bigint "course_from_find_id"
+    t.integer "course_from_find_id"
```

Comment:
Is this intentional, guess not?

I think I had exactly the same thing happen once and ended up resetting my dev db. Maybe a postgres version thing.

Reply:
I believe its a version thing, according to the migration files this should be `integer` but a previous migration changed https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/commit/8e3dcb555bfdc2fd9432de7744bd4ff8899b96f9

we discussed this before with @tijmenb and it should be harmless

Reply:
What is your `psql --version` @davidgisbey ?

Reply:
I'm on 10.10 but I've not upgraded for ages